lib/rules/html-indent.js

```diff
@@ -349,11 +353,19 @@ function create (context) {
           setOffset(baseToken, offset, leftToken)
         }
 
-        // Align the rest tokens to the first token.
+        // Set baseline
         if (nodeList.some(isBeginningOfLine)) {
           setBaseline(baseToken)
         }
-        setOffset(alignTokens, 0, baseToken)
+
+        if (!options.alignAttributesVertically && rootNode) {
```

Comment:
Would you move `options.alignAttributesVertically` to the call site? `processNodeList` is a common logic for many expressions/statements. I'd like to change the value of `rootNode` by the option.